sparkplug/timereporters/base.py

```diff
@@ -0,0 +1,43 @@
+import datetime
+
+##################################################
+
+def _milliseconds(timedelta):
+    return timedelta / datetime.timedelta( microseconds=1000 )
+
+
+##################################################
+
+def mn_md_mx(x):
+    "calculate min median max from a list of timedeltas, returns the time in milliseconds"
```

Comment:
This should be a comment, not a string.

Reply:
What is the argument against a docstring?

Reply:
The general pytohn convention of a docstring is to start/end with 3 quotes (eg.  `"""` or `'''`).